### PR TITLE
feature/CORE-549/CORE-662/test-advanced-block-configs-to-mutate-state

### PR DESCRIPTION
**Pull request by bulatgab**
Demo:

https://github.com/FLOIP/flow-builder/assets/18130716/82b5918b-0f39-437a-859d-94c67b925832

### CODE REVIEW

File: src/components/interaction-designer/block-types/Core_LogBlock.vue [hunk 1]
```diff
@@ -23,6 +23,7 @@
             :placeholder="'flow-builder.enter-message' | trans"
             :current-expression="value"
             :valid-state="isValid"
+            data-cy="log-block--message--input"
```
Comment (safydy-r) [inline src/components/interaction-designer/block-types/Core_LogBlock.vue:26]: any particular reason we prefixed with `log-block--` ?

File: cypress/e2e/undo-redo/block-configs-should-update-flow-state/advanced-block-configs.cy.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    cy.get('[data-cy="run-flow-block--flow-id--input"]')
+      .find('textarea')
+      .type(blockConfigs.flowId)
+
+    firstBlockState().its('config.flow_id').should('eq', blockConfigs.flowId)
+  })
+})
+
+const getStore = () => cy.window().its('store')
```
Comment (safydy-r) [inline cypress/e2e/undo-redo/block-configs-should-update-flow-state/advanced-block-configs.cy.ts:56]: good idea to extract these 👍

**Review by safydy-r — APPROVED** [review]
LGTM

File: src/components/interaction-designer/block-types/Core_LogBlock.vue (hunk 1 above)
Comment (bulatgab) [reply src/components/interaction-designer/block-types/Core_LogBlock.vue]: You're right, it was not necessary. Renamed!

**Pull request merged**